spec/features/visitor/sign_in_spec.rb

```diff
@@ -29,4 +30,10 @@ def sign_in(email, password)
 
     expect(page).to have_content("You have to confirm your email address before continuing.")
   end
+
+  scenario "js object stores Visitor data", js: true do
```

Comment:
@ruslankhaertdinov I think this part of js should be tested somewhere else. May be we could use `jasmin` specs for that?